src/pages/benchmarks/index.tsx

```diff
@@ -142,6 +142,20 @@ function Benchmarks() {
               */
               }
 
+              {/* simple sort pill POC (see TODO above) */}
+              <button
```

Comment:
For a future PR: This should be a <PillButton> component with props for sortCriterion ("created_at") and title ("Sort by date")

Reply:
cool, will dev in Storybook as discussed